index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
```

Comment:
I can see that you have not used the provided Microverse README template. Microverse has a recommended template that students use for all their projects in Microverse. Kindly use this template to create a professional README.md file https://github.com/microverseinc/readme-template. For this template attached, kindly do the following when customizing it to suit this project:

#### **_Please note that where ever you see these square brackets - ( [some text] )anywhere in the README file, you should edit the text and remove the brackets afterward._**

_**Also, note that all section in the README is compulsory except the FAQ section. Kindly edit the template accordingly.**_

- [ ] Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it. 🙏
- [ ] Please add your project title.
- [ ] For the Tech Stack Section, Kindly remove the Server and Database subsections because the technologies listed there are not relevant to this project. For the Client section, kindly include HTML and CSS 👍
- [ ] For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. 👍
- [ ] For the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:

    -  Installation of Git
    -  Installation of NPM
    -  A working browser
    -  Internet connection
    -  A code editor like Visual Studio Code
    -  Installation of Live Server plugging in Visual Studio Code
    -  Any other tool you can think about

- [ ] For the Setup section, kindly add instructions for cloning your repository on a local machine 👍
- [ ] For the Install section, kindly add instructions for setting up linters used in this project
- [ ] For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.
- [ ] For the **_Author_** section, please edit it with your GitHub and social media handles.
- [ ] For the MIT section, kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE - https://choosealicense.com/licenses/mit/

**_Kindly implement all the things listed above. It will make your README.md file look professional and it will give you an idea of how to set README.md files for most of the projects you will execute as a Microverse student._** 👍 💻